src/deploy/functions/runtimes/golang/index.ts

```diff
@@ -141,6 +145,14 @@ export class Delegate {
     });
   }
 
+  // eslint-disable-next-line
+  async discoverBuild(
+    configValues: backend.RuntimeConfigValues,
+    envs: backend.EnvironmentVariables
+  ): Promise<build.Build> {
+    return { requiredAPIs: [], endpoints: {}, params: [] };
```

Comment:
nit* can we add a comment to explain that this fn is unimplemented? we can also just throw an error 🤷🏼

Reply:
Done.